.jenkins/staticlibrary.groovy

```diff
@@ -0,0 +1,82 @@
+#!/usr/bin/env groovy
+@Library('rocJenkins@pong') _
+import com.amd.project.*
+import com.amd.docker.*
+import java.nio.file.Path;
+
+def runCI =
+{
+    nodeDetails, jobName->
+    
+    def prj = new rocProject('hipCUB', 'Static Library PreCheckin')
+
+    def nodes = new dockerNodes(nodeDetails, jobName, prj)
+
+    def commonGroovy
+
+    boolean formatCheck = false
+     
+    def compileCommand =
+    {
+        platform, project->
+
+        commonGroovy = load "${project.paths.project_src_prefix}/.jenkins/common.groovy"
+        commonGroovy.runCompileCommand(platform, project, jobName, false)
```

Comment:
commonGroovy.runCompileCommand(platform, project, jobName, false, true)
Needs true at the end